deye_config.py

```diff
@@ -59,10 +63,10 @@ def __init__(self, host: str, port: int, username: str, password: str, topic_pre
     def from_env():
         return DeyeMqttConfig(
             host=os.getenv('MQTT_HOST'),
-            port=int(os.getenv('MQTT_PORT')),
+            port=int(os.getenv('MQTT_PORT', '1883')),
             username=os.getenv('MQTT_USERNAME'),
             password=os.getenv('MQTT_PASSWORD'),
-            topic_prefix=os.getenv('MQTT_TOPIC_PREFIX'),
+            topic_prefix=os.getenv('MQTT_TOPIC_PREFIX', 'deye'),
```

Comment:
These changes are not related to the scope of the PR

Reply:
Agree that this are not strictly related. I added this changes since my Python highlighted them as possible issue in case of unset env variable (no handling of None) and I thought that having a default is reasonable in this case.
Should I remove this changes?

Reply:
Okay, let's keep that